packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.kt

```diff
@@ -718,7 +797,17 @@ class FileDownloader(
     .header("Expo-API-Version", "1")
     .header("Expo-Updates-Environment", "BARE")
     .header("EAS-Client-ID", easClientID)
-    .header("Accept", if (allowPatch && assetEntity.isLaunchAsset && configuration.enableBsdiffPatchSupport) "$PATCH_CONTENT_TYPE,*/*" else "*/*")
+    .apply {
+      val shouldRequestPatch = allowPatch && assetEntity.isLaunchAsset && configuration.enableBsdiffPatchSupport
+      val currentId = extraHeaders.optString(EXPO_CURRENT_UPDATE_ID_HEADER, "")
+      val requestedId = extraHeaders.optString(EXPO_REQUESTED_UPDATE_ID_HEADER, "")
+      header("Accept", "application/javascript")
```

Comment:
I think we want to keep this `*/*`? (to match current implementation since we're no longer using the `Accept` header for this)
```suggestion
      header("Accept", "*/*")
```